Runtime/Scripts/DeviceInfoIntegration.cs

```diff
@@ -45,20 +62,22 @@ enum NetworkDataType
     public Dictionary<string, string> GetDeviceInfo()
     {
       CarrierInfoClass carrierInfo = new CarrierInfoClass();
-      AndroidJavaObject telephonyManager = carrierInfo.GetTelephonyManager();
       Dictionary<string, string> map;
-
+      int sdk_int = carrierInfo.getAndroidSDKVers();
+      if (UnityEngine.XR.XRSettings.isDeviceActive && UnityEngine.XR.XRSettings.loadedDeviceName.Contains("oculus"))
```

Comment:
Ok. Once we have a better idea what DeviceInfo needs to be, this will be good enough. No telephony layer to get anything from at all.